java/client/test/org/openqa/selenium/support/ui/FluentWaitTest.java

```diff
@@ -172,7 +172,8 @@ public void timeoutMessageIncludesLastIgnoredException() {
   @Test
   public void timeoutMessageIncludesCustomMessage() {
     TimeoutException expected = new TimeoutException(
-      "Timed out after 0 seconds: Expected custom timeout message");
+        "Could not match predicate: Expected custom timeout message "
```

Comment:
What do you think about "Expected condition failed: " ? Would like to keep as much focus on the condition as possible.